object.js

```diff
@@ -96,12 +110,39 @@ function hint(gj, options) {
                     line: _.__line__ || line
                 });
             }
+            if (_.length > 3) {
+                return errors.push({
+                    message: 'position should not have more than 3 elements',
+                    line: _.__line__ || line
+                });
+            }
             if (!everyIs(_, 'number')) {
                 return errors.push({
                     message: 'each element in a position must be a number',
                     line: _.__line__ || line
                 });
             }
+
+            // This is an arbitrary threshold roughly corresponding to
+            // the upper limit of survey-grade GPS precision in CRS84
+            // anything more and we're into microscopy
+            var maxPrecision = 9;
+            var num;
+            for (var i = 0; i < _.length; i++) {
+                num = _[i];
+                // TODO there has got to be a better way. Check original text?
+                // By this point number has already been parsed to a float...
+                var precision = 0;
+                var decimalStr = (num + "").split(".")[1]
```

Comment:
This project has an extremely lenient eslint that just catches errors, rather than style issues.